tests/conftest.py

```diff
@@ -91,14 +122,25 @@ def hlwm_process(tmpdir):
 
     proc = subprocess.Popen([bin_path, '--verbose'], env=env,
                             stdout=subprocess.PIPE)
+    hlwm_proc = HlwmProcess(proc)
```

Comment:
So the `HlwmProcess` wraps around the return value of `Popen`, right? Why not integrate the `Popen` into its constructor? I was a bit confused by this contraction between the class name and its functionality.

Reply:
Right, that sounds more reasonable. I just tried to make the diff as small as possble.